docs/groovy-script/mods/mwc/crafting_station.md

```diff
@@ -0,0 +1,209 @@
+    .output(item('minecraft:diamond'))
+    .register()
+
+mods.mwc.crafting_station.recipeBuilder()
+    .input(0.5, item('minecraft:gold_ingot'))
+    .output(item('minecraft:clay') * 2)
+    .register()
+```
+
+:::::::::
+
+::::::::::
+
+## Removing Recipes
+
+- Remove all recipes of Bullet Category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeAllBullet()
+    ```
+
+- Remove all recipes of Grenade Category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeAllGrenade()
+    ```
+
+- Remove all recipes of Gun Category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeAllGun()
+    ```
+
+- Remove all recipes of Magazine Category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeAllMagazine()
+    ```
+
+- Remove all recipes of Modification Attachment Category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeAllModificationAttachment()
+    ```
+
+- Remove all recipes of Normal Attachment Category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeAllNormalAttachment()
+    ```
+
+- Removes recipe that outputs matching item of given `ingredient` from bullet category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeBulletByOutput(IIngredient)
+    ```
+
+- Removes recipe that outputs matching item of given `ingredient` from grenade category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeGrenadeByOutput(IIngredient)
+    ```
+
+- Removes recipe that outputs matching item of given `ingredient` from gun category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeGunByOutput(IIngredient)
+    ```
+
+- Removes Recipe that outputs given `ingredient` from given category. valid categories = "GUN", "ATTACHMENT_NORMAL", "ATTACHMENT_MODIFICATION", "BULLET", "MAGAZINE", "GRENADE":
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeInGroupByOutput(IIngredient, CraftingGroup)
+    ```
+
+- Removes recipe that outputs matching item of given `ingredient` from magazine category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeMagazineByOutput(IIngredient)
+    ```
+
+- Removes recipe that outputs matching item of given `ingredient` from modification attachment category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeModificationAttachmentByOutput(IIngredient)
+    ```
+
+- Removes recipe that outputs matching item of given `ingredient` from normal attachment category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeNormalAttachmentByOutput(IIngredient)
+    ```
+
+- Removes all recipes that match the given output:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeByOutput(IIngredient)
+    ```
+
+- groovyscript.wiki.mwc.crafting_station.removeAllInGroup:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeAllInGroup(CraftingGroup)
```

Comment:
would fit the standard pattern if it was renamed to be `removeByGroup`. 
additionally, probably want a `removeByGroup(String)` (`removeAllInGroup(String)`) method to go along with this.